docs/Training-SAC.md

```diff
@@ -0,0 +1,348 @@
+# Training with Soft-Actor Critic
+
+In addition to [Proximal Policy Optimization (PPO)](Training-PPO.md), ML-Agents provides
+[Soft Actor-Critic](http://bair.berkeley.edu/blog/2018/12/14/sac/) to perform
+reinforcement learning.
+SAC uses several neural networks. During training, it uses two neural networks to learn
```

Comment:
"Sac uses several neural networks: - this statement seems a bit out of place

Reply:
also, u explain it in the next sentence so maybe this isn't needed.

Reply:
I decided to remove this paragraph as it seems like unneeded info.